### PR TITLE
Display type information when type alias is hovered in rbs

### PR DESCRIPTION
**Pull request by Yuuki77**
we now display type information when alias is hovered in rbs 
image

## Known issue
The judgment when hovering is off by one character.
I think this is a location issue, so I'll fix it in a later PR.

### CODE REVIEW

File: lib/steep/server/interaction_worker.rb [hunk 1]
```diff
@@ -80,9 +80,16 @@ def process_hover(job)
             content = hover.content_for(path: job.path, line: job.line, column: job.column+1)
             if content
               range = content.location.yield_self do |location|
-                start_position = { line: location.line - 1, character: location.column }
-                end_position = { line: location.last_line - 1, character: location.last_column }
-                { start: start_position, end: end_position }
+                case content.location
```
Comment (soutaro) [inline lib/steep/server/interaction_worker.rb:83]: You can use `#as_lsp_range` (https://github.com/soutaro/steep/blob/master/lib/steep/range_extension.rb)

Comment (Yuuki77) [reply lib/steep/server/interaction_worker.rb]: thank you for the review!
updated it here https://github.com/soutaro/steep/pull/385/commits/d93aee587ef7710723f6f5791b8af0989d8682b7

**Review by soutaro — APPROVED** [review]
👍

**Comment by soutaro** [comment]
@Yuuki77 One of the tests is failing...

**Comment by Yuuki77** [comment]
@soutaro sorry I just fixed the test
it should be ok now 💪

**Pull request merged**